stdlib/Test/src/Test.jl

```diff
@@ -179,8 +182,7 @@ function Base.show(io::IO, t::Error)
         println(io, "  Test threw exception")
         println(io, "  Expression: ", t.orig_expr)
         # Capture error message and indent to match
-        print(io, join(map(line->string("  ",line),
-                           split(t.backtrace, "\n")), "\n"))
+        join(io, ("  " * line for line in split(t.backtrace, "\n")), "\n")
```

Comment:
true, it could similarly also avoid the regex with `print(io, replace(t.backtrace, '\n' => "\n  "))`. I don't know which is better, I was just making the current one more concise and avoid temporaries.

Reply:
That's what I initially thought, but then the first line doesn't get indented. No big deal anyway.